t/wasm/global-rule.t

```diff
@@ -57,13 +57,6 @@ __DATA__
             local code, body = t('/apisix/admin/global_rules/1',
                 ngx.HTTP_PUT,
                 [[{
-                    "uri": "/hello",
-                    "upstream": {
-                        "type": "roundrobin",
-                        "nodes": {
-                            "127.0.0.1:1980": 1
-                        }
-                    },
```

Comment:
Why do you remove this?

Reply:
This field is not as expected and is not allowed to be passed into upstream in the global-rule.

Reply:
If you delete this block, the test case will fail

Reply:
I have test this PR on myself repo https://github.com/Sn0rt/apisix/pull/9